client/src/+assets/checkout.tsx

```diff
@@ -1,89 +1,113 @@
-                          <UiText variants={['secondary']}>{defi.category}</UiText>
-                          <UiText variants={['detail']}>&nbsp;&ndash;&nbsp;</UiText>
-                          <UiText variants={['secondary']}>{defi.subcategory}</UiText>
+                          <UiIcon className={b('info-icon')} icon="tag" color="secondary" />
+                          <UiText variants={['secondary']}>
+                            <XuiTokenPrice>{metadata.main.price}</XuiTokenPrice>
+                            {' '}
+                            <UiText variants={['detail']}>
+                              <XuiTokenName address={getDdoTokenAddress(asset)} />
+                            </UiText>
+                          </UiText>
                         </UiLayout>
-                      </>
-                    )}
-                    <UiLayout className={b('info')}>
-                      <UiIcon className={b('info-icon')} icon="tag" color="secondary"/>
-                      <UiText variants={['secondary']}>
-                        <XuiTokenPrice>{metadata.main.price}</XuiTokenPrice>
-                        {' '}
-                        <UiText variants={['detail']}>
-                          <XuiTokenName address={getDdoTokenAddress(asset)}/>
-                        </UiText>
-                      </UiText>
-                    </UiLayout>
-                  </UiLayout>
-                </div>
-              ))
+                      </UiLayout>
+                    </div>
+                  ))
+                :
+                <UiLayout type='grid' justify='center' >
+                  No item in your basket
+                </UiLayout>
+
             }
+
           </div>
           <div className={b('order-summary')}>
             <UiText type="h3" wrapper="h2">Order summary</UiText>
-            <hr style={{ borderBottom: '0', borderTop: '1px solid #8AAABE', margin: '22px 0 16px', width: '426px' }}/>
+            <hr style={{ borderBottom: '0', borderTop: '1px solid #8AAABE', margin: '22px 0 16px', width: '426px' }} />
             <UiLayout justify="end"><div className={b('items-selected')}>{basket.length} items</div></UiLayout>
-            <UiDivider/>
+            <UiDivider />
             <UiLayout justify="space-between">
               <UiText type="h4-caps" wrapper="h2">TOTAL</UiText>
               <div className={b('total-price')}><XuiTokenPrice>{totalPrice}</XuiTokenPrice></div>
             </UiLayout>
-            <UiDivider/>
-            <XuiCreateBundle assets={assets}>
-            { assets.length > 0 ? <UiButton cover>Purchase</UiButton> : <></>}
-            </XuiCreateBundle>
-            <UiDivider/>
+            <UiDivider />
+            {
+              !(isLogged && account) ?
+                <UiButton cover onClick={loginMetamask}>Connect Wallet</UiButton>
+                :
+                (network === correctNetworkName && assets.length > 0 && connected) ?
```

Comment:
with `assets.length` instead of `assets.length > 0`

Reply:
🆗